LayoutTests/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/track/track-element/track-remove-by-setting-innerHTML.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!DOCTYPE html><!-- webkit-test-runner [ captionDisplayMode=automatic ] -->
```

Comment:
Please do not use `<!-- webkit-test-runner` directives in WPT

Reply:
It is needed for webvtt related tests. :-( Although in this case, it is not, so I will not remove it.